components/MashForms/ScreeningDecisionForm.spec.tsx

```diff
@@ -0,0 +1,98 @@
+import { render, fireEvent, screen, waitFor } from '@testing-library/react';
+import { mockedMashReferral } from 'factories/mashReferral';
+import ScreeningDecisionForm from './ScreeningDecisionForm';
+import { submitScreeningDecision } from 'utils/api/mashReferrals';
+import { mockedWorker } from 'factories/workers';
+
+jest.mock('utils/api/mashReferrals');
+
+const mockPush = jest.fn();
+jest.mock('next/router', () => ({
+  useRouter: () => ({
+    push: mockPush,
+  }),
+}));
+
+describe('#ScreeningDecisionForm', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+
+    render(
+      <ScreeningDecisionForm
+        referral={mockedMashReferral}
+        workerEmail={mockedWorker.email}
+      />
+    );
+  });
+
+  it('should render correctly', () => {
+    expect(screen.getByText('Make screening decision'));
+    expect(screen.getByText('Document the decision'));
+    expect(screen.getByText('Select screening decision'));
+    expect(screen.getByText('Is this contact urgent?'));
+    expect(screen.getByText('Submit'));
+  });
+
+  it('should dynamically render hint text for emailing a mash manager if decision is urgent', () => {
+    expect(
+      screen.queryByText(
+        'Please email your MASH manager about the urgent case.'
+      )
+    ).toBeNull();
+
+    fireEvent.click(screen.getByText('Yes'));
+
+    expect(
+      screen.getByText('Please email your MASH manager about the urgent case.')
+    );
+  });
+
+  it('should disable submit button when submitScreeningDecision Submit button is clicked', () => {
```

Comment:
Does mean that the button is disabled after it is clicked? Why do we need to do this?

Reply:
We don't want a user clicking the button multiple times sending multiple API calls. 

It also indicates to the user that their submission is being sent as disabling the button modifies it's appearance.

Before a user could click, see no indication that clicking did anything,  click again. It wouldn't break anything but this is better UX and stops the API potentially getting spammed.

Reply:
I see 🙌🏾 . Do we show a notification/banner on the page? Might also help the user understand what's going on too?